apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.Snippets/project.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
     "Google.Cloud.Diagnostics.AspNet": { "target": "project" },
-    "xunit": "2.2.0-rc3-build3528"
+    "xunit": "2.2.0-rc4-build3536"
```

Comment:
Is there something in this version of xunit that you need? If so, I'm happy to perform a *global* find/replace, then this PR can be on top of that. It would cut down the number of affected files, making it simpler.

If this was just a matter of "update to latest just in case" I'd prefer to leave them out.